test/scraper/hessen_test.rb

```diff
@@ -0,0 +1,91 @@
+require 'test_helper'
+
+class HessenTest < ActiveSupport::TestCase
+  def setup
+    @scraper = HessenScraper
+    @overview = Nokogiri::HTML(File.read(Rails.root.join('test/fixtures/hessen_overview.html')))
+    @detail_minor = Nokogiri::HTML(File.read(Rails.root.join('test/fixtures/hessen_detail_minor.html')))
+    @detail_major = Nokogiri::HTML(File.read(Rails.root.join('test/fixtures/hessen_detail_major.html')))
+    @search = Nokogiri::HTML(File.read(Rails.root.join('test/fixtures/hessen_search.html')))
+    @blocks = @scraper.extract_blocks @overview
+    @detail_block_minor = @scraper.extract_detail_block(@detail_minor)
+    @detail_block_major = @scraper.extract_detail_block(@detail_major)
+    @search_result = @scraper.extract_result_from_search(@search)
+  end
+
+  test 'get overview blocks' do
+    blocks = @scraper.extract_blocks @overview
+    assert_equal 634, blocks.length
+  end
+
+  test 'get detail block' do
+    assert_equal true, @scraper.extract_detail_block(@detail_minor).content.include?('und Antw 17.02.2015 Drs')
+    assert_equal true, @scraper.extract_detail_block(@detail_major).content.include?('23.02.2015 Drs 19/1632')
+  end
+
+  test 'get reference from block' do
+    assert_equal ['19', '1774'], @scraper.extract_reference(@blocks.first)
+  end
+
+  test 'extract_interpellation_type' do
+    assert_equal Paper::DOCTYPE_MINOR_INTERPELLATION, @scraper.extract_interpellation_type(@blocks.first)
+  end
+
+  test 'extract title' do
+    assert_equal 'Studium Generale', @scraper.extract_title(@blocks.first)
+  end
+
+  test 'extract originators' do
+    text = @scraper.extract_originator_text(@detail_block_minor)
+    assert_equal({people: ["Marius Weiß", "Norbert Schmitt", "Wolfgang Decker", "Kerstin Geis", "Brigitte Hofmeyer", "Gerald Kummer", "Angelika Löber", "Torsten Warnecke"], parties: ["SPD"]}, @scraper.extract_originators(text))
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.